internal/aez/general.go

```diff
@@ -21,6 +21,7 @@ type STAT struct { //(34 octets)
 	SPWEEP uint32 // SpaceWire received EEPs
 }
 
-func (s *STAT) read(buf io.Reader) error {
-	return binary.Read(buf, binary.BigEndian, s)
+// Read STAT
+func (stat *STAT) Read(buf io.Reader) error {
+	return binary.Read(buf, binary.BigEndian, stat)
```

Comment:
how about a test of a real STAT buffer from a `.rac` file resulting in the expected STAT struct?